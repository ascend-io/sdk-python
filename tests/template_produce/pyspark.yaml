id: pyspark
component:
  transform:
    operator:
      sparkFunction:
        executable:
          code:
            source:
              inline: |-
                def transform(inputs):
                    return inputs[0]
