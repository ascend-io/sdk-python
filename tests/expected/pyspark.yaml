# id: pyspark
{%- set ascend_pyspark = ascend_include_raw(resourceParentDirectory, "pyspark_pyspark.py") %}
component:
  transform:
    operator:
      sparkFunction:
        executable:
          code:
            source:
              inline: {{ ascend_pyspark | dumpJson }}
